pkg/feature/features.go

```diff
@@ -36,5 +41,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[feature.Feature]feature.FeatureSpec{
-	ValidateCAA: {Default: false, PreRelease: feature.Alpha},
+	ValidateCAA:   {Default: false, PreRelease: feature.Alpha},
+	DisableTmpCrt: {Default: false, PreRelease: feature.Alpha},
```

Comment:
Can we rename this to `IssueTemporaryCertificate`, set Default to `true` and `PreRelease` `feature.Beta`? 😄